robots/centauro_description/urdf/centauro.urdf

```diff
@@ -0,0 +1,1398 @@
+    </inertial>
+    <visual>
+      <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
+      <geometry>
+        <mesh filename="package://example-robot-data/robots/centauro_description/meshes/simple/wrist_pitch_right.stl" scale="0.001 -0.001 0.001"/>
+      </geometry>
+    </visual>
+    <collision>
+      <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
+      <geometry>
+        <mesh filename="package://example-robot-data/robots/centauro_description/meshes/simple/wrist_pitch_right.stl" scale="0.001 -0.001 0.001"/>
+      </geometry>
+    </collision>
+  </link>
+  <joint name="j_arm1_1" type="revolute">
+    <parent link="torso_2"/>
+    <child link="arm1_1"/>
+    <origin rpy="0.174533 0.0 -0.5235988" xyz="0.0457475 0.169137 0.116626"/>
+    <axis xyz="0 1 0"/>
+    <limit effort="147.0" lower="-3.312" upper="1.615" velocity="3.86"/>
+  </joint>
+  <!-- /shoulder yaw -->
+  <!-- shoulder roll -->
+  <joint name="j_arm1_2" type="revolute">
+    <parent link="arm1_1"/>
+    <child link="arm1_2"/>
+    <origin rpy="-0.1745327 0.0 0.0" xyz="-0.09015 0.062 0.0"/>
+    <axis xyz="1 0 0"/>
+    <limit effort="147.0" lower="0.02" upper="3.431" velocity="3.86"/>
+  </joint>
+  <!-- /shoulder roll -->
+  <!-- shoulder pitch-->
+  <joint name="j_arm1_3" type="revolute">
+    <parent link="arm1_2"/>
+    <child link="arm1_3"/>
+    <origin xyz="0.09015 0.0 -0.21815"/>
+    <axis xyz="0 0 1"/>
+    <limit effort="147.0" lower="-2.552" upper="2.566" velocity="6.06"/>
+  </joint>
+  <!-- /shoulder pitch -->
+  <!-- elbow yaw -->
+  <joint name="j_arm1_4" type="revolute">
+    <parent link="arm1_3"/>
+    <child link="arm1_4"/>
+    <origin xyz="0.045 0.05515 -0.074"/>
+    <axis xyz="0 1 0"/>
+    <limit effort="147.0" lower="-2.465" upper="0.28" velocity="6.06"/>
+  </joint>
+  <!-- /elbow yaw -->
+  <!-- elbow pitch-->
+  <joint name="j_arm1_5" type="revolute">
+    <parent link="arm1_4"/>
+    <child link="arm1_5"/>
+    <origin xyz="-0.015 -0.05515 -0.095"/>
+    <axis xyz="0 0 1"/>
+    <limit effort="55.0" lower="-2.569" upper="2.562" velocity="11.72"/>
+  </joint>
+  <!-- /elbow pitch-->
+  <!-- wrist yaw-->
+  <joint name="j_arm1_6" type="revolute">
+    <parent link="arm1_5"/>
+    <child link="arm1_6"/>
+    <origin xyz="0.0 0.049 -0.156"/>
+    <axis xyz="0 1 0"/>
+    <limit effort="55.0" lower="-1.529" upper="1.509" velocity="11.72"/>
+  </joint>
+  <link name="arm2_1">
+    <inertial>
+      <origin xyz="-0.0074457212 -0.03410796 0.00010978102"/>
+      <mass value="1.9628675"/>
+      <inertia ixx="0.0053547717" ixy="-0.00036428926" ixz="1.5089568e-05" iyy="0.0033923328" iyz="-5.5692312e-05" izz="0.0068921413"/>
+    </inertial>
+    <visual>
+      <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
+      <geometry>
+        <mesh filename="/root/ibrido_ws/src/iit-centauro-ros-pkg/centauro_urdf/meshes/ShoulderPitch.STL" scale="0.001 0.001 0.001"/>
```

Comment:
These paths are wrong, ref CI:
`py-example-robot-data> message: Could not load resource /root/ibrido_ws/src/iit-centauro-ros-pkg/centauro_urdf/meshes/v2/collision/hip-yaw.stl`